### DISCUSSION TITLE
If not stock Intel AC 8265 WLAN, then which one for AirDrop & Continuity?

### DISCUSSION THREAD

**Discussion by bobmarley2021** [valnoxy/t480-oc#113] [Q&A]
Hi, sorry if this is a silly question but I am limping along on a Broadcom card (Dell DW1830 / BCM43602) but I get absolutely terrible 5Ghz wifi performance. If I use the stock Intel AC 8265 I get great performance, but I lose Airdrop (continuity still works as does the shared clipboard though which is odd!)

Under the Readme section "What's not working" it says:  AirDrop & Continuity (**only devices with Intel WiFi**) but [this discussion](https://github.com/valnoxy/t480-oc/discussions/32) says the native card won't work with AirDrop.

So my question is, which Intel wifi card should I get? I am running Ventura not Sonoma... is it that the stock Intel AC 8265 card works better in Sonoma? Or do I just need a different card. If it's a case of it working in Sonoma, I'll continue limping. But if another Intel card will work better I'd rather just grab that.

Any help greatly appreciated :)

**Comment by ellnic** [comment]
I was wondering this too. I am also using the Dell DW1830, I wouldn't say the speeds are bad, but the Intel is definitely quicker.

I don't have exact figures right now but I think my previous speed tests were about 450Mbit for Intel and about half that for the Broadcom. Note: I didn't do proper testing using iperf, just a quick check using Ookla. I'm on Gigabit FTTP so no bottleneck there.

I use Airdrop a lot so have stuck with the Broadcom card for now.

This must be a hardware issue or hack issue because looking at the spec:

Dell DW1830 info: http://en.techinfodepot.shoutwiki.com/wiki/Dell_Wireless_1830_(DW1830)
Intel AC 8265 info: https://www.intel.com/content/www/us/en/products/sku/94150/intel-dual-band-wirelessac-8265/specifications.html

Unless I'm mistaken the Broadcom card is better on paper with the exception of BLE being version 4.1 vs 4.2 for Intel.

@bobmarley2021 what is "absolutely terrible" to you?
@valnoxy you have the Dell DW1830, what 5Ghz speeds do you get?

**Comment by bobmarley2021** [comment]
Absolutely terrible is 20-25mbit! 😞 it's usable for normal day to day stuff, even a bit of streaming but nothing like large network transfers or downloads. Like I said, I'm limping along but would like to fix in the future if at all possible.

**Comment by ellnic** [comment]
Wow.... you might have a faulty card there, that is shocking. Let's see what @valnoxy says about his speeds before making a judgement but that seems like hardware failure to me. At the very least one of your antennas has an issue.

**Comment by chxbw** [comment]
I suggest you upgrade directly to Intel's AX210. I tested this card on WiFi at home, and if I remember correctly, it can run almost the maximum limit of WiFi. I am quite certain about this, and this card is not that expensive. 8265AC is indeed a bit old. I suggest you use Intel's AX210 directly, and I think DW1560 should be better? Or 943602CS? I am currently macOS 13 and using the AX210


If it looks strange to you, it's because I'm using translation software...
